### ISSUE TITLE
flag parsing in master seems broken

### ISSUE THREAD

**Issue by bacongobbler** [kubernetes/helm#2688]
On a fresh build from master this morning, I am seeing the following issue whenever a flag is invoked:

```
><> helm-dev version --client
unknown flag: --client
Usage of homer:
      --home string   location of your Helm config. Overrides $HELM_HOME (default "$HOME/.helm")
Client: &version.Version{SemVer:"v2.5+unreleased", GitCommit:"1261f71faf1aa8b9a127a6e1759fd799a51d9528", GitTreeState:"clean"}
><> helm-dev search --versions local
unknown flag: --versions
Usage of homer:
      --home string   location of your Helm config. Overrides $HELM_HOME (default "$HOME/.helm")
NAME      	VERSION    	DESCRIPTION                
local/test	0.1.0-rc.10	A Helm chart for Kubernetes
local/test	0.1.0-rc.9 	A Helm chart for Kubernetes
local/test	0.1.0-rc.1 	A Helm chart for Kubernetes
```

When no flags are present, no error is displayed.

```
><> helm-dev version
Client: &version.Version{SemVer:"v2.5+unreleased", GitCommit:"1261f71faf1aa8b9a127a6e1759fd799a51d9528", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.5.0", GitCommit:"012cb0ac1a1b2f888144ef5a67b8dab6c2d45be6", GitTreeState:"clean"}
```

Not sure if this is fixed by #2682 or not, but I figured I'd open a new ticket for visibility/tracking purposes.

**Comment by bacongobbler** [comment]
Confirmed with @adamreese that #2682 is the fix.

**Comment by bacongobbler** [comment]
closed via #2690
